pyroomacoustics/libroom_src/ray_tracing.cpp

```diff
@@ -0,0 +1,50 @@
+#include <Eigen/Dense>
```

Comment:
It doesn't look like this file is used. We can remove it.

Reply:
Done.